runtime/compiler/ilgen/Walker.cpp

```diff
@@ -5568,8 +5568,8 @@ TR_J9ByteCodeIlGenerator::loadFromCP(TR::DataType type, int32_t cpIndex)
                                                             comp());
                   if (primitiveCondyCriticalSection.hasVMAccess())
                      {
-                     uintptrj_t* objLocation = (uintptrj_t*)_methodSymbol->getResolvedMethod()->dynamicConstant(cpIndex);
-                     uintptrj_t obj = *objLocation;
+                     uintptrj_t obj = 0;
+                     uintptrj_t* objLocation = (uintptrj_t*)_methodSymbol->getResolvedMethod()->dynamicConstant(cpIndex, obj);
```

Comment:
Below, when doing `loadConstant(TR::iconst, *(int32_t*)(obj+valueOffset));` we can run into trouble for JITServer. Maybe that `tryToAcquireVMAccess` is implemented such that it always fails on JITServer